integration-tests/init.spec.js

```diff
@@ -14,15 +14,14 @@ const fs = require('fs')
 const DD_INJECTION_ENABLED = 'tracing'
 const DD_INJECT_FORCE = 'true'
 const DD_TRACE_DEBUG = 'true'
-
+const { NODE_VERSION } = require('../version')
 const telemetryAbort = ['abort', 'reason:incompatible_runtime', 'abort.runtime', '']
 const telemetryForced = ['complete', 'injection_forced:true']
 const telemetryGood = ['complete', 'injection_forced:false']
 
 const { engines } = require('../package.json')
 const supportedRange = engines.node
-const currentVersionIsSupported = semver.satisfies(process.versions.node, supportedRange)
-
+const currentVersionIsSupported = semver.satisfies(NODE_VERSION, supportedRange, process.version.node)
```

Comment:
```suggestion
const currentVersionIsSupported = semver.satisfies(NODE_VERSION, supportedRange, process.version.node)

```

Preserve existing spacing untouched.